portfolio/src/main/webapp/style.css

```diff
@@ -90,6 +90,18 @@ body{
     width: 33.33%;
 }
 
+@media screen and (max-width: 1800px) {
```

Comment:
You may also be interested in units of measurement in terms of viewport width -- "vw". So,  `width: 50vw` would be half of visible width.

https://developer.mozilla.org/en-US/docs/Web/CSS/length#Viewport-percentage_lengths